README.md

```diff
@@ -16,4 +16,20 @@ docker run -e CRAPI_URL=http://host.docker.internal:8090 -e HTTP_PROXY=http://ho
 ./gradlew buildDockerImage
 export CRAPI_URL=<crapi url>
 docker-compose up
-```
\ No newline at end of file
+```
+
+
+
+**Step By Step Cypress Installation**
+1. Install Node.js. After Installation:
+  ## In MAC: 
+          Go to Terminal, echo $PATH, /usr/local/bin should be present in the path. If not present, vim ~/.bash_profile and click enter
+          Click "i" to insert. export PATH=$PATH:/usr/local/bin and save using(esc:wq) and click enter.
+          To save changes globally, source ~/.bash_profileand enter
+  ## In Windows: 
+          Open System environment variables, Under System Variables. Click New. Add Variable name:NODE_HOME and value:C:\Program Files\nodejs and OK
+2. Download Visual studio Code and install
+3. Clone the repo
+4. In VS Code Terminal, enter npm install(Installs all dependencies in package.json)
```

Comment:
Navigate to `app/` and run `npm install`.

Reply:
Resolved